libs/dh/dev-examples/feature-processes/src/details/details.component.ts

```diff
@@ -0,0 +1,212 @@
+//#region License
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+//#endregion
+import { afterRenderEffect, Component, computed, inject, input, viewChild } from '@angular/core';
+
+import { TranslocoDirective, TranslocoPipe } from '@ngneat/transloco';
+
+import {
+  WattDescriptionListComponent,
+  WattDescriptionListItemComponent,
+} from '@energinet-datahub/watt/description-list';
+
+import {
+  WattProgressTrackerComponent,
+  WattProgressTrackerStepComponent,
+} from '@energinet-datahub/watt/progress-tracker';
+
+import { WattDatePipe } from '@energinet-datahub/watt/date';
+import { VaterFlexComponent } from '@energinet-datahub/watt/vater';
+import { WATT_DRAWER, WattDrawerComponent } from '@energinet-datahub/watt/drawer';
+
+import { lazyQuery } from '@energinet-datahub/dh/shared/util-apollo';
+import { DhProcessStateBadge } from '@energinet-datahub/dh/wholesale/shared';
+import { DhNavigationService } from '@energinet-datahub/dh/shared/navigation';
+import { GetProcessDocument } from '@energinet-datahub/dh/shared/domain/graphql';
+import { DhEmDashFallbackPipe, DhResultComponent } from '@energinet-datahub/dh/shared/ui-util';
+
+import type {
+  DhProcessCalculation,
+  DhProcessWholesaleRequest,
+  DhProcessEnergyTimeSeriesRequest,
+} from '../types';
+
+import { DhCalculationsDetailsGridAreasComponent } from './gridareas.component';
+
+@Component({
+  imports: [
+    TranslocoPipe,
+    TranslocoDirective,
+
+    WATT_DRAWER,
+    WattDatePipe,
+    WattProgressTrackerComponent,
+    WattDescriptionListComponent,
+    WattDescriptionListItemComponent,
+    WattProgressTrackerStepComponent,
+
+    VaterFlexComponent,
+
+    DhResultComponent,
+    DhProcessStateBadge,
+    DhEmDashFallbackPipe,
+    DhCalculationsDetailsGridAreasComponent,
+  ],
+  selector: 'dh-process-details',
+  styles: `
+    .headline {
+      display: inline-block;
+      padding-right: var(--watt-space-s);
+    }
+
+    dh-calculation-details-grid-areas {
+      margin: 0 var(--watt-space-ml);
+    }
+
+    watt-progress-tracker {
+      margin: 0 var(--watt-space-ml);
+    }
+  `,
+  template: `
+    @let process = result();
+    @let calculation = calculationDetails();
+    @let energyTimeSeriesRequest = energyTimeSeriesRequestDetails();
+    @let wholesaleRequest = wholesaleRequestDetails();
+
+    <watt-drawer *transloco="let t; read: 'devExamples.processes'" (closed)="closed()">
```

Comment:
suggestion: maybe inline the contents of `closed()`?

Reply:
I agree a single line is good to inline fixed in https://github.com/Energinet-DataHub/greenforce-frontend/pull/3958/commits/20e4154dee826ad501630d27de7f5d1c09a04d6e